source/Applications/Advanced/HandEyeCalibration/README.md

```diff
@@ -13,26 +13,27 @@ To fully understand Hand-Eye Calibration, please see the [tutorial](https://zivi
    4. Repeat 1.-3. until 10-20 pose pairs are collected
    5. Enter command to perform calibration and return a **Transformation Matrix**
 
-[ZividHandEyeCalibration](C:\Program Files\Zivid\bin\ZividHandEyeCalibration.exe): (no source)
+[ZividHandEyeCalibration](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/96469274)
 
-* Application which takes a collection of pose pairs (e.g. output of steps 1.-3. in [SampleHandEyeCalibration]([SampleHandEyeCalibration-url])) and returns a **Transformation Matrix**
+* CLI application which takes a collection of pose pairs (e.g. output of steps 1.-3. in [HandEyeCalibration][HandEyeCalibration-url]) and returns a **Transformation Matrix**
 
 -----------------
 The following applications assume that a **Transformation Matrix** has been found
 
-[**UtilizeEyeInHandCalibration**]([UtilizeEyeInHandCalibration-url]):
+[**UtilizeEyeInHandCalibration**][UtilizeEyeInHandCalibration-url]:
 
 * Shows how to transform position and rotation (pose) in Camera co-ordinate system to Robot co-ordinate system.
 * Example use case - "Bin Picking":
    1. Acquire point cloud of objects to pick with Zivid camera
    2. Find optimal picking pose for object and **transform to robot co-ordinate system**
    3. Use transformed pose to calculate robot path and execute pick
 
-[**PoseConversions**]([PoseConversions-url]):
+[**PoseConversions**][PoseConversions-url]:
 
 * Zivid primarily operate with a (4x4) Transformation Matrix (Rotation Matrix + Translation Vector). This example shows how to use Eigen to convert to and from:
   * AxisAngle, Rotation Vector, Roll-Pitch-Yaw, Quaternion
 
-[SampleHandEyeCalibration-url]: https://www.zivid.com/hubfs/softwarefiles/releases/1.6.0+7a245bbe-26/doc/cpp/zivid_sample_code.html#autotoc_md10
-[UtilizeEyeInHandCalibration-url]: https://github.com/zivid/zivid-cpp-samples/blob/master/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
-[PoseConversions-url]: https://github.com/zivid/zivid-cpp-samples/blob/master/Applications/Advanced/HandEyeCalibration/PoseConversions/PoseConversions.cpp
\ No newline at end of file
+[HandEyeCalibration-url]: HandEyeCalibration/HandEyeCalibration.cpp
+[UtilizeEyeInHandCalibration-url]: UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
+[ZividHandEyeCalibration-url]: file:///C:/Program&#32;Files/Zivid/bin/ZividHandEyeCalibration.exe
```

Comment:
This seems weird. What is file:///C:/Program&#32?

Reply:
Yeah this looks wrong. The variable is never user either.

Reply:
Read online that I needed to add "&#32;" where a link normally has space. But im not using that link anyway, so I removed it